personalsite/src/App.js

```diff
@@ -0,0 +1,137 @@
+import './App.css';
+import React from 'react';
+import './App.css';
+import Navbar from "./components/Navbar/Navbar";
+import LocationPosting from './page2';
+import Page3 from './page3';
+import { Component } from 'react';
+import axios from 'axios'; 
+
+
+
+
+const buttonFunction = () => {
+  console.log("the button was clicked");
+  var x = document.getElementById("myDIV");
+  if (x.style.display === "none") {
+  x.style.display = "block";
+  }
+  else {
+  x.style.display = "none";
+  }
+}
+
+const swimming = {
+  name: "Swimming",
+  years: 10
+}
+
+const diving = {
+  name: "Diving",
+  years: 4 
+}
+
+const waterpolo = {
+  name: "Waterpolo",
+  years: 3 
+}
+
+const mySports = [swimming, diving, waterpolo];
+const sportMap = mySports.map((sport) => {
+  return(
+  <div className = "sport">
+      <p>
+        {sport.name} for {sport.years} years.
+      </p>
+    </div>
+  )
+  })
+
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state={
+      currPage: "Home",
+      data: null
+    }
+    }
+
+    fetchData = () => {
```

Comment:
nice job incorporating the poetry api! There are tons of free apis that you can use if youre interested in practicing with another!